demo/KevinDemo/Stories/Payment/PaymentView.swift

```diff
@@ -0,0 +1,221 @@
+                                        },
+                                        textBinding: $viewModel.viewState.amountString,
+                                        text: viewModel.viewState.amountString,
+                                        keyboardType: .decimalPad
+                                    )
+                                    
+                                    Text("kevin_window_main_amount_field_currency".localized())
+                                        .style(.currencyHint)
+                                        .padding(.trailing, 14.0)
+                                }
+                            }
+                            
+                            KevinAgreementCheckMark(
+                                isAgreementChecked: viewModel.viewState.isAgreementChecked,
+                                toggleAgreement: {
+                                    viewModel.toggleAgreement()
+                                }
+                            )
+                            
+                            Button {
+                                hideKeyboard()
+                                viewModel.openPaymentTypeSelection()
+                            } label: {
+                                Text(String(
+                                    format: "kevin_window_main_proceed_button".localized(),
+                                    viewModel.viewState.amountString == "" ? "0.00" : viewModel.viewState.amountString,
+                                    "kevin_window_main_amount_field_currency".localized())
+                                ).style(.buttonTitle)
+                                    .frame(minWidth: 0, maxWidth: .infinity)
+                            }
+                            .buttonStyle(MainButtonStyle())
+                            .disabled(viewModel.viewState.isDonateButtonDisabled)
+                        }
+                        .frame(
+                            minWidth: 0,
+                            maxWidth: .infinity,
+                            minHeight: 0,
+                            maxHeight: .infinity,
+                            alignment: .topLeading
+                        )
+                        .padding(16)
+                    }
+                }
+                .navigationBarTitle("kevin_window_main_title_label".localized(), displayMode: .large)
+            }
+        }
+        .halfASheet(
+            isPresented: $viewModel.viewState.isCountrySelectorPresented,
+            content: {
+                KevinCountrySelector(
+                    countryCodes: viewModel.viewState.countryCodes,
+                    selectedCountry: viewModel.viewState.selectedCountryCode!,
+                    onCountrySelected: { selectedCountryCode in
+                        viewModel.selectCountry(selectedCountryCode)
+                    }
+                )
+            },
+            configuration: HalfASheetConfiguration(
```

Comment:
Those configurations look similar maybe make a property or function to generate them?